packages/linear-genome-view/src/LinearGenomeView/components/RubberBand.tsx

```diff
@@ -68,8 +68,8 @@ function RubberBand({
   ControlComponent?: React.ReactElement
   children: React.ReactNode
 }) {
-  const [startX, setStartX] = useState()
-  const [currentX, setCurrentX] = useState()
+  const [startX, setStartX] = useState<number>()
```

Comment:
since it is initialized as undefined, templating it as just number is technically incorrect here

Reply:
should be number|undefined

Reply:
the resulting type of `startX` ends up being `number | undefined` there, so I think React already puts in the undefined by itself